terraform/projects/app-ecs-services/config/alerts/registers.yml

```diff
@@ -9,3 +9,22 @@ groups:
     annotations:
         summary: "App {{ $labels.app }} has too many 5xx errors"
         description: "App {{ $labels.app }} has 5xx errors in excess of 25% of total requests"
+
+  - alert: Registers_DiskSpaceExceeded
+    expr: disk_utilization{org="openregister",space="prod"} > 80
+    for: 300s
+    labels:
+        product: "registers"
+    annotations:
+        summary: "App {{ $labels.app }} has exceeded 80% disk space usage for 5 minutes"
+        description: "TODO"
+
+  - alert: Registers_CPUExceeded
+    expr: avg_over_time(cpu{org="openregister",app="beta-multi",space="prod"}[5m])
```

Comment:
Same comment here as above regarding use of `space` rather than `exported_space`